plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveAnalyzeCorruptStatistics.java

```diff
@@ -119,5 +120,14 @@ private void prepareBrokenColumnStatisticsTable(String tableName)
             executor.shutdownNow();
             executor.awaitTermination(10, SECONDS);
         }
+
+        // Verify whether column statistics in metastore is broken or not
+        try {
```

Comment:
optional: move this logic to `testAnalyzeCorruptColumnStatisticsOnEmptyTable()`

Reply:
Disagreed as the method name is `prepareBrokenColumnStatisticsTable`. A broken table should exist after called the method.